rita_client/src/operator_update/ops_websocket.rs

```diff
@@ -229,14 +289,19 @@ fn get_five_minute_update_data(
         client_mbps,
         relay_mbps,
     };
-    let serialized = serde_json::to_vec(&data).unwrap();
+    let encrypted = encrypt_router_websocket_msg(id.wg_public_key, our_secretkey, ops_pubkey, data);
```

Comment:
you can have the encrypted messages implement a .json() call that does the serde_json::to_vec then you could just call encrypt_router_websocket_msg(args).json()